### ISSUE TITLE
How to divide the training set, validation set, test set

### ISSUE THREAD

**Issue by Chinjohn** [VisDrone/DroneCrowd#3]
How to divide the training set, validation set, test set？？
And your code don't have make_data.py。。。
